src/com/company/Contact.java

```diff
@@ -0,0 +1,23 @@
+package com.company;
+
+public class Contact {
+    String name;
+    String telephone;
+    String email;
+
+    Contact(String nameContact, String telephoneContact, String emailContact){
+        name=nameContact;
```

Comment:
@SFDC-Melby-Arguedas When your are refering to class attribute, do it with **this** keyword like: 
_**this**.name = nameContact_;
Also you need space between equals.